Bukkit/src/main/java/com/plotsquared/bukkit/listener/HighFreqBlockEventListener.java

```diff
@@ -167,6 +170,18 @@ public void onPhysicsEvent(BlockPhysicsEvent event) {
             plot.debug("Prevented block physics and resent block change because disable-physics = true");
             return;
         }
+        if (plot.getFlag(DisableBlockUpdateFlag.class)) {
+            if (!event.getChangedType().hasGravity()) {
+                if (sourceBlock.getType().isAir() || block.getType().isAir()) {
+                    return;
+                }
+                if (!sourceBlock.getType().name().toLowerCase(Locale.ROOT).contains("snow")
+                        || !block.getLocation().add(0, -1, 0).getBlock().getType().equals(Material.GRASS_BLOCK)) {
```

Comment:
I still think this is a massively specific implementation. It should be an "all or nothing" which is why I question its inclusion; this might be how _you_ want it, but it's not necessarily how others might want it to be implemented. This is why I suggest it might make better sense in a separate plugin where it is configurable

Reply:
For which practical creative construction use-case was this implemented? I would also say that “no block update” does not need any filter.